project_task_material/__manifest__.py

```diff
@@ -1,16 +1,17 @@
-# -*- coding: utf-8 -*-
 # Copyright 2012 - 2013 Daniel Reis
 # Copyright 2015 - Antiun Ingeniería S.L. - Sergio Teruel
 # Copyright 2016 - Tecnativa - Vicent Cubells
 # Copyright 2017 - Tecnativa - David Vidal
+# Copyright 2018 - Brain-tec AG - Carlos Jesus Cebrian
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Project Task Material",
     "summary": "Record products spent in a Task",
-    "version": "10.0.1.0.0",
+    "version": "11.0.1.0.0",
     "category": "Project Management",
     "author": "Daniel Reis,"
               "Tecnativa,"
+              "Braintec,"
```

Comment:
This migration doesn't represent too much work, so you should add as contributor, but not being promoted to co-author.